utils/bin/pause-hive-sync

```diff
@@ -28,14 +28,12 @@ _logged_in(){
 }
 
 # Cluster deployment must be provided
-while getopts ":hlc:s:" opt; do
+while getopts ":hc:s:" opt; do
   case ${opt} in
     h ) _help ; exit 0
       ;;
     c ) CLUSTER_DEPLOYMENT=${OPTARG}
       ;;
-    l ) LIST="true"
```

Comment:
I couldn't find anywhere in the entire project that this was used, but please correct me if I'm missing something and I can easily add it back.

Reply:
I don't believe it's being used anymore either.  Must have been cruft from the past.